.travis.yml

```diff
@@ -8,11 +8,13 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
   - hhvm
   
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7.3
```

Comment:
I think this shouldn't be allowed

Reply:
I allowed it to fail as 7.3 isn't released yet.